### PR TITLE
British drug cos

### LINKED ISSUE #62: AstraZeneca, Loughborough, UK

### PR DESCRIPTION
**Pull request by sbillinge**
closes #62, #63

### CODE REVIEW

File: db/institutions.yml [hunk 1]
```diff
@@ -369,20 +371,13 @@ uiuc:
...
-  - northwestern
-  city: Evanston
-  name: Northwestern University
-  state: IL
-  zip: '60208'
 ghentu:
-  aka: [University of Ghent, Universiteit Gent, UGent, State University of Ghent, Rijksuniversiteit Gent, RUG]
+  aka: [University of Ghent, Universiteit Gent, UGent, State University of Ghent,
+    Rijksuniversiteit Gent, RUG]
   city: Ghent
```
Comment (chiahaoliu) [inline db/institutions.yml]: I've always wanted to learn the indentation rule in json. Could you please explain when we need 2 spaces (like here) and when we need 6 spaces as the aka filed inside department dict (line 312 in this PR)? Thanks so much.

File: db/institutions.yml [hunk 2]
```diff
@@ -305,14 +305,16 @@ qmul:
...
   country: Mexico
-  schools: {sciences: {name: Faculty of Sciences, aka: [Facultad de Ciencias]}}
-  departments: {physics: {name: Department of Physics, aka: [Departamiento de física, Departamiento de fisica]}}
+  departments: {physics: {name: Department of Physics, aka: [Departamiento de física,
+        Departamiento de fisica]}}
   name: National Autonomous University of Mexico
-  street: República de Brasil 33
+  schools: {sciences: {name: Faculty of Sciences, aka: [Facultad de Ciencias]}}
   state: ''
+  street: República de Brasil 33
```
Comment (sbanerjee23) [inline db/institutions.yml]: This address doesn't seem to be right for UNAM uni center, or the Faculty of Sciences or the dept of physics. I think you should use this (Coyoacán is the state):
```
Av Universidad 3000, Cd. Universitaria, Coyoacán, 04510 Ciudad de México, CDMX, Mexico
```
image

Comment (sbanerjee23) [reply db/institutions.yml]: Otherwise all the drug company entries looked fine to me.

**Comment by sbillinge** [comment]
Thanks @sbanerjee23 .  The UNAM entry wasn't mine (I must have done some reformat on the entry for it to show up), but I made the modification and pushed, it was an excellent catch!  nice job.  

Please review again and pull in if it looks ok.  Thanks.

**Comment by sbanerjee23** [comment]
Looks good to me!

**Comment by sbillinge** [comment]
Thanks @sbanerjee23 !

**Pull request merged**